_posts/2022-05-02-the-na-bug/the-na-bug.Rmd

```diff
@@ -0,0 +1,111 @@
+---
+title: "Ooh na na... where are my sodium labs?"
+description: |
+  The NA bug, or, what happens when the same word is used in different contexts.
+author:
+  - name: Chloe Pou-Prom
+    affiliation: DSAA, Unity Health Toronto
+    affiliation_url: https://chartdatascience.ca
+date: 2022-05-02
+output:
+  distill::distill_article:
+    self_contained: false
+categories:
+  - language-R
+  - project-chartwatch
+  - post-miscellaneous
```

Comment:
For categories, I was thinking we could follow the convention where each blog post should have at least 3 categories:
- The language: e.g., `language-R`, `language-python`, `language-javascript`
- The project: e.g., `project-chartwatch`, `project-rushh`, `project-hemo`
- The type of post: e.g., `post-journal-club`, `post-extended-standup`, `post-miscellaneous`,.......... more?
- Other categories could for example be the package/library name (e.g., `funneljoin`, `modelstudio`) or a data-science topic (e.g., `anomaly-detection`, `dei`)